sklearn/svm/libsvm.pyx

```diff
@@ -133,23 +133,23 @@ def fit(
     Returns
     -------
     support : array, shape=[n_support]
-        index of support vectors
+        Index of support vectors.
 
     support_vectors : array, shape=[n_support, n_features]
-        support vectors (equivalent to X[support]). Will return an
+        Support vectors (equivalent to X[support]). Will return an
         empty array in the case of precomputed kernel.
 
-    n_class_SV : array
-        number of support vectors in each class.
+    n_class_SV : array, shape=[n_class]
```

Comment:
should we not go with what's suggested in https://github.com/scikit-learn/scikit-learn/pull/12356 ?

Reply:
I should be able to fix this. I need sometime to go through #12356 though.